Samples/LineGraph/README.md

```diff
@@ -0,0 +1,81 @@
+
+## Preparation
+
+In order to use the library, you have to install it. This is done by running the following command from the commandline:
+
+```
+uno install Fuse.Charting
+```
+
+With that done, you should be all set.
+
+## The line-graph renderer
+
+The line-graph renderer should function as a `ux:Class` we can place inside a `Plot` element to have a graph drawn the way we want it. We are therefore interrested in adding a few properties so we get more control over the graph:
+
+	<Panel ux:Class="MyPlot">
+		<int ux:Property="SeriesIndex" />
+		<float4 ux:Property="MainColor" />
+		<float4 ux:Property="BackgroundColor" />
+
+One property of interest, is `SeriesIndex`. If you provide multiple `DataSeries` to a `Plot`, you can use the `SeriesIndex` on a `PlotData` element to select which data series you are drawing from. By exposing the `SeriesIndex` as a property, followed by reading from it locally using `ReadProperty`(see below), we can control what data series our renderer is reading from.
+
+Our line-graph renderer should take a `DataSeries`, and display the data in a nice and consistent way. We can do this using `PlotData`. `PlotData` is like `Each`, but works on every point of data in the `Plot` we are drawing on. Using this, we can place our dot and curve elements in the UX hierarchy, and further draw a graph:
+
+
+	<Curve StrokeWidth="2" StrokeColor="{ReadProperty MainColor}" Close="None">
+		<c:PlotData SeriesIndex="{ReadProperty SeriesIndex}">
+			<c:PlotCurvePoint />
+			<c:PlotPoint>
+				<Circle Color="{ReadProperty MainColor}" Width="10" Height="10">
+					<Stroke Width="2" Color="#202B33" Alignment="Outside" />
+				</Circle>
+			</c:PlotPoint>
+		</c:PlotData>
+	</Curve>
+
+Notice how the `PlotData` element is inside a `Curve` element. This is because we want to repeat PlotCurvePoints inside a curve, in order to build its shape. As `PlotPoint` doesn't affect a `Curve`, it is safe to place that in the same location, which is why we also place the dot here.
+
+In addition to drawing a curve from our plot data, we want to extrude it downwards. This can be done using the `Extrude` property on `Curve`, like so:
+
+	<Curve Close="None" Extrude="Bottom">
+		<SolidColor Color="{ReadProperty BackgroundColor}" />
+		<c:PlotData SeriesIndex="{ReadProperty SeriesIndex}">
+			<c:PlotCurvePoint />
+		</c:PlotData>
+	</Curve>
+
+## Putting it together
+
+With a line-graph renderer done, we can make a `Plot` element, and use it to pipe some test data to three line-graph renderers:
+
+	<c:Plot >
+		<c:DataSeries Data="{values1}" />
+		<c:DataSeries Data="{values2}" />
+		<c:DataSeries Data="{values3}" />
+
+		<GridLayout Rows="1*,20" />
+
+		<Panel Row="0">
+			<c:PlotTicks Axis="Y" StrokeWidth="1" StrokeColor="#d0d0d019" />
+		</Panel> 
+
+		<MyPlot Row="0" MainColor="#48B66A" BackgroundColor="#213536" SeriesIndex="2" />
```

Comment:
please format this code better and place them in the correct order according to series index (unless that ordering is important).